test_rclcpp/test/test_multiple_service_calls.cpp

```diff
@@ -70,3 +70,56 @@ TEST(CLASSNAME(test_multiple_service_calls, RMW_IMPLEMENTATION), multiple_servic
   printf("Received second reply\n");
   EXPECT_EQ(2, result2.get()->sum);
 }
+
+TEST(CLASSNAME(test_multiple_service_calls, RMW_IMPLEMENTATION), multiple_service_calls) {
+  uint32_t n = 5;
+
+  rclcpp::init(0, nullptr);
+
+  auto node = rclcpp::Node::make_shared("test_multiple_service_calls");
+
+  node->create_service<test_rclcpp::srv::AddTwoInts>(
+    "test_multiple_service_calls", handle_add_two_ints);
+
+  using ClientRequestPair =
+      std::pair<rclcpp::client::Client<test_rclcpp::srv::AddTwoInts>::SharedPtr,
+      test_rclcpp::srv::AddTwoInts::Request::SharedPtr>;
```

Comment:
I find these two lines hard to read. What about the following which makes the pair more visible:

    using ClientRequestPair = std::pair<
        rclcpp::client::Client<test_rclcpp::srv::AddTwoInts>::SharedPtr,
        test_rclcpp::srv::AddTwoInts::Request::SharedPtr>;